### DISCUSSION TITLE
How to change the width of the border around the controls on the card?

### DISCUSSION THREAD

**Discussion by gogenzuller** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#203] [Q&A] [answered]
Greetings. Please help me to change the controls-wrapper margin from 15 to 10px. If its value is 15, then when I open the card in a Chrome on my smartphone, the elements do not fit in one row. I tried to change it in 1407 row of `/www/community/lovelace-xiaomi-vacuum-map-card/xiaomi-vacuum-map-card.js` but nothing changed.
![Screenshot 2021-11-22 111820](https://user-images.githubusercontent.com/10682203/142829509-372bd761-5179-4621-b3dc-d1fd6a0c3c6d.jpg)

**Comment by PiotrMachowski** [comment, marked as answer]
You can do it using [card-mod](https://github.com/thomasloven/lovelace-card-mod):
```yaml
style: |
  .controls-wrapper {
    margin: 10px !important;
  }
```

**Reply by gogenzuller** [reply]
Thanks, it works.
